go/tasks/plugins/array/k8s/monitor.go

```diff
@@ -224,7 +224,7 @@ func FetchPodStatusAndLogs(ctx context.Context, client core.KubeClient, name k8s
 			o, err := logPlugin.GetTaskLogs(tasklog.Input{
 				PodName:          pod.Name,
 				Namespace:        pod.Namespace,
-				LogName:          fmt.Sprintf(" #%d-%d", index, retryAttempt),
+				LogName:          fmt.Sprintf(" #%d-%d", retryAttempt, index),
```

Comment:
Don't they get ordered by the name? is this better UX than essentially grouping the task by the index?

Reply:
This is cleaner as this is in-sync with the pod name which can be otherwise confusing if you are looking at k8s-logs or just investigating any issues.

Reply:
agreed